app/uk/gov/hmrc/ivorchestration/handlers/IvSessionDataRequestHandler.scala

```diff
@@ -51,7 +50,7 @@ class IvSessionDataRequestHandler[F[_]](
     }
 
   protected def generateIdAndPersist(ivSessionData: IvSessionData): F[IvSessionDataCore] =
-    persist(IvSessionDataCore(ivSessionData, JourneyId(UUID.randomUUID().toString), new DateTime))
+    persist(IvSessionDataCore(ivSessionData, JourneyId(UUID.randomUUID().toString), LocalDateTime.now()))
```

Comment:
We've switched from Joda to Java time - that's fine, but we need to be really careful we end up using the same serialization format.

Reply:
Might be better to just do the Mongo upgrade for now?  And leave Joda conversion for another ticket?